src/serac/physics/utilities/variational_form/detail/hexahedron_hcurl.inl

```diff
@@ -1,7 +1,7 @@
 template <int p>
-struct finite_element<::Geometry::Hexahedron, Hcurl<p>> {
-  static constexpr auto geometry   = ::Geometry::Hexahedron;
-  static constexpr auto family     = ::Family::HCURL;
+struct finite_element<Geometry::Hexahedron, Hcurl<p>> {

```

Comment:
I think I ran into issues using `Geometry` rather than `::Geometry` when a sourcefile had `using namespace mfem`, since they also define `Geometry`. I'm not sure if that issue will still exist after putting these definitions in the serac namespace. e.g. if I have a .cpp file with

```cpp
using namespace mfem
using namespace serac

auto g = Geometry::Hexahedron;
```
will it still work?

Reply:
So we still wouldn't be able to have that snippet work unless we disambiguated the two somehow (renamed the enum in serac, or specified the namespace explicitly, etc)